README.md

```diff
@@ -10,6 +10,17 @@ Testing
 Put files to be populated by Ignition in the tree rooted at
 `data/ignition/files/`. Put systemd units in `data/ignition/systemd/units`.
 
+The agent.service file requires SERVICE_BASE_URL and INFRA_ENV_ID to be set.
+The pull secret is also required and is written to /root/.docker/config.json in
+the ISO.
+For now, these are set through environment variables.
+
+```shell
+export PULL_SECRET=$(cat ~/Downloads/pull-secret.txt)
+export SERVICE_BASE_URL=http://10.0.1.10:6000
```

Comment:
`192.168.122.2:8090` or `127.0.0.1:8090`

Reply:
The above suggestion is not needed after https://github.com/openshift-agent-team/fleeting/pull/5